### PR TITLE
Hmrc header

### PR DESCRIPTION
**Pull request by nataliecarey**

### CODE REVIEW

**Comment by nataliecarey** [comment]
This header includes an optional sign-out link - to see the preview `npm start` on this branch and visit http://localhost:3000/components/header/sign-out-link/preview

File: app/app.js [hunk 1]
```diff
@@ -32,6 +32,7 @@ module.exports = (options) => {
 
   // make the function available as a filter for all templates
   env.addFilter('componentNameToMacroName', helperFunctions.componentNameToMacroName)
+  env.addFilter('componentNameToComponentDirectory', helperFunctions.componentNameToComponentDirectory)
```
Comment (rpowis) [inline app/app.js]: Do we need this as filter?

File: src/components/hmrc-header/_header.scss [hunk 2]
```diff
@@ -0,0 +1,298 @@
+@import "../../settings/all";
```
Comment (rpowis) [inline src/components/hmrc-header/_header.scss]: all these styles should be coming from govuk-frontend so i don't think we need this file

File: src/components/hmrc-sign-out-link/README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Sign out link
```
Comment (rpowis) [inline src/components/hmrc-sign-out-link/README.md]: The sign out link doesn't need to be a separate component.

File: app/app.js [hunk 4]
```diff
@@ -68,7 +69,7 @@ module.exports = (options) => {
     const components = fileHelper.allComponents
 
     res.render('index', {
-      componentsDirectory: components
+      componentsDirectory: components.map(x => x.match(/^(?:hmrc-)?(.*)$/)[1])
```
Comment (rpowis) [inline app/app.js]: Can we define the department prefix in config?

**Review by rpowis — COMMENTED** [review]
Looking good! Just a couple comments

**Pull request merged**